node/_fs/_fs_write.js

```diff
@@ -0,0 +1,36 @@
+import { Buffer } from "../buffer.ts";
+
+export function writeSync(fd, bufferLike, ...args) {
+  const [buffer, pos] = bufferAndPos(bufferLike, args);
+  if (typeof pos === "number") {
+    Deno.seekSync(fd, pos, Deno.SeekMode.Start);
+  }
+  return Deno.writeSync(fd, buffer);
+}
+
+export function write(fd, bufferLike, ...args) {
+  const cb = args[args.length - 1];
+  const [buffer, pos] = bufferAndPos(bufferLike, args.slice(0, -1));
+
+  async function innerWrite() {
+    if (typeof pos === "number") {
+      await Deno.seek(fd, pos, Deno.SeekMode.Start);
+    }
+    return await Deno.write(fd, buffer);
```

Comment:
This should probably use `writeAll()` - there's no guarantee that the whole buffer will be written in one go